libs/eo/auth/routing-security/src/lib/eo-authentication.guard.ts

```diff
@@ -37,14 +40,18 @@ export class EoScopeGuard implements CanActivate {
     return this.authStore.getScope$.pipe(
       map((scope) => {
         if (scope.length === 0) {
-          this.router.navigate(['/login'], {
-            queryParams: { redirectionPath: window.location.pathname + window.location.search },
-          });
+          window.location.assign(
+            `${window.location.protocol}//${window.location.host}/${this.transloco.getActiveLang()}?redirectionPath=${window.location.pathname}${window.location.search}`
+          );
```

Comment:
Hvorfor denne ændring?

Reply:
Det skyldes at vi nu har to SPA's som skal navigere imellem hinanden. For at komme ud af konteksten fra den ene app og over i den anden kan man ikke længere bruge angular router.